bool camera_node_rdy